recipes/heatshrink/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.36.0"
+
+class HeatshrinkConan(ConanFile):
+    name = "heatshrink"
+    license = "ISC"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "data compression library for embedded/real-time systems"
+    topics = ("compression", "embedded", "realtime")
+    homepage = "https://github.com/atomicobject/heatshrink"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+    options = {
+        "shared": [False, True],
+        "fPIC": [True, False],
+        "dynamic_alloc": [True, False],
+        "debug_log": [True, False],
+        "use_index": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "dynamic_alloc": True,
+        "debug_log": False,
+        "use_index": True,
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+
+        config_file = os.path.join(self._source_subfolder, "heatshrink_config.h")
```

Comment:
Please move patching to `build`.
```suggestion
    def _patch_sources(self):
        config_file = os.path.join(self._source_subfolder, "heatshrink_config.h")
```